.vscode/settings.json

```diff
@@ -36,5 +36,8 @@
         "${workspaceFolder}/cpp/IceStorm/counter",
         "${workspaceFolder}/cpp/IceStorm/replicated",
         "${workspaceFolder}/cpp/IceStorm/replicated2",
-    ]
+    ],
+    "cmake.configureSettings": {
```

Comment:
This just tells the Ice CMake file to print its configuration during configuration. It will appear in the Visual Studio CMake console.